src/index.html

```diff
@@ -22,6 +22,382 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="Images/logo.png"
+          alt="MoYo Logo"
```

Comment:
Ensure the image path is correct. Usually, image folders are named in lowercase, such as 'images' instead of 'Images'.